src/grafana/chart/templates/uds-package.yaml

```
@@ -28,23 +28,20 @@ spec:
         targetPort: 3000
 
     allow:
-      - direction: Ingress
+      # Egress allowed to Loki
+      - direction: Egress
         selector:
           app.kubernetes.io/name: grafana
-        remoteNamespace: tempo
+        remoteNamespace: loki
         remoteSelector:
-          app.kubernetes.io/name: tempo
-        port: 9090
-        description: "Tempo Datasource"
+          app.kubernetes.io/name: loki
+        description: "for loki"
 
+      # Egress allowed to Prometheus
       - direction: Egress
         selector:
           app.kubernetes.io/name: grafana
-        remoteGenerated: Anywhere
-
-      - direction: Egress
-        remoteNamespace: tempo
+        remoteNamespace: monitoring
         remoteSelector:
-          app.kubernetes.io/name: tempo
-        port: 9411
-        description: "Tempo"
+          app.kubernetes.io/name: prometheus
+        description: "for monitoring"
```

very nit, but might prefer this description to be "Prometheus Datasource"
```suggestion
        description: "Prometheus Datasource"
```

Current netpol name for reference:
```
allow-grafana-egress-for-monitoring                      app.kubernetes.io/name=grafana   8s
```

(on this one, if you don't agree with this suggestion would suggest at least changing to `prometheus` instead of `monitoring`)

---

I think we could also lock this policy down to a specific port - potentially port 9090 (https://github.com/defenseunicorns/uds-core/blob/lock-down-egress/src/grafana/chart/templates/datasources.yaml#L16)